### ISSUE TITLE
Wrong encoding on IntellijIDEA 2016.2

### ISSUE THREAD

**Issue by ibnuda** [tonsky/FiraCode#246]
First of all, I'm sorry if my issue / question is misplaced.
When I use FiraCode regular on IntellijIDEA 2016, I got the following encoding.
![firacode_intellij_ligature_off](https://cloud.githubusercontent.com/assets/8169908/17649146/09878692-6257-11e6-84a8-f76e324b7cc0.PNG)
And it doesn't matter whether I use ligature or not, the result is the same. But a couple days ago, everything was just fine.

So, my question is : 
1. Why the font looks like that?
2. How can I solve this problem?

My system is : Windows 10 64bit.

Thankyou for your help.

**Comment by tonsky** [comment]
I got this too last time I tried to install Fira on Win10! But apparently
other people have it working. Can you report to that to
https://youtrack.jetbrains.com/?

**Comment by ibnuda** [comment]
Yes. I will report it to Jetbrains and close this issue.
Thanks for your time.

**Comment by tonsky** [comment]
Issue in question https://youtrack.jetbrains.com/issue/IDEA-159901

**Comment by tonsky** [comment]
Ok, strange thing just happened. I used to have the same exact issue with corrupted everything on my Windows 10 with 2016.2 and just checked it was still there with 2016.3 EAP. It happened on both 32-bit bundled and 64-bit Oracle JVMs, on both 1.200 and 1.201 TTF versions of Fira Code. So I went into `C:\Windows\Fonts` with `cmd`, did `dir` there only to find out that there’re 6 hidden (!) TTF files with names like `FiraCode-Regular_2.ttf`. These were not visible with Explorer. So I del them all, then installed Fira Code 1.201 TTFs from distr and everything went to normal. No more corruption. I wonder what that was.
